Satuju/ReusableView/DetectLanguageButton.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  DetectLanguageButton.swift
+//  Satuju
+//
+//  Created by Gede Wicaksana on 02/11/21.
+//
+
+import SwiftUI
+
+struct DetectLanguageButtonView: View {
+    var body: some View {
+        Button {
+        } label: {
+            Label(
+                title: {},
+                icon: { Image("micIcon")
+                        .resizable()
+                        .frame(width: 27, height: 41, alignment: .center)
+                        .foregroundColor(.white)
+                        .frame(width: 66, height: 66, alignment: .center)
+                        .background(LinearGradient(gradient: Gradient(colors: [Color("AquaSecondary"), Color("PurplePrimary")]), startPoint: .top, endPoint: .bottom))
```

Comment:
```suggestion
                        .background(LinearGradient(gradient: Gradient(colors: [Color("AquaSecondary"), Color("PurplePrimary")]), startPoint: .top, endPoint: .bottom))
```
ini melanggar batas karakter dalam satu line, coba kayak gini
```
.background(
       LinearGradient(
                 gradient: Gradient(
                        colors: [Color("AquaSecondary"), Color("PurplePrimary")]
                  ),
                  startPoint: .top, endPoint: .bottom
        )
  )
```